testsuite/forge-cli/src/main.rs

```diff
@@ -1112,10 +1112,10 @@ fn realistic_env_workload_sweep_test() -> ForgeConfig {
         ]),
         // Investigate/improve to make latency more predictable on different workloads
         criteria: [
-            (7700, 100, 0.3, 0.3, 0.5, 0.5),
-            (7000, 100, 0.3, 0.3, 0.5, 0.5),
-            (2000, 300, 0.3, 0.8, 0.6, 0.7),
-            (3200, 500, 0.3, 0.4, 0.7, 1.0),
+            (7700, 100, 0.3, 0.5, 0.5, 0.5),
+            (7000, 100, 0.3, 0.5, 0.5, 0.5),
+            (2000, 300, 0.3, 1.0, 0.6, 1.0),
+            (3200, 500, 0.3, 1.5, 0.7, 1.0),
```

Comment:
last column, I swapped them, the 4th line should be 0.7

are you increasing pos-> prop column because of this PR? Because other than first line (which seems more like an outlier), rest were all kept below 0.3

Reply:
Yes, the pos->prop is changed by the doing less backpressure (some txn queueing shifts from mempool to pos->prop)

Oh I see, I'll make the last line 0.7

Reply:
change made